Sources/SwiftDocC/Model/Rendering/RenderSectionTranslator/ParametersSectionTranslator.swift

```diff
@@ -20,15 +20,27 @@ struct ParametersSectionTranslator: RenderSectionTranslator {
         translateSectionToVariantCollection(
             documentationDataVariants: symbol.parametersSectionVariants
         ) { _, parameters in
-            guard !parameters.parameters.isEmpty else { return nil }
+            guard !(parameters.parameters.isEmpty || parameters.parameters.count < 1) else {
+                return ParameterRenderSection(name: "", content: []) as? RenderSection
```

Comment:
And why not return `nil` here? `RenderNodeTranslator#createRenderSections` calls this, which will drop `nil` sections using `compactMap` which seems fine to me.

Reply:
You're right, just tested this again and it's working! I'm really not sure what was happening, but earlier it was not compiling when I was just returning `nil` which is why I made this change. Now when I run a clean build it's working and all the tests are still passing, so I must've forgotten to clean or something??